docs/source/stats-collection.rst

```diff
@@ -0,0 +1,72 @@
+.. _stats_collection:
+
+Comparing Spider Executions
+===========================
+
+Sometimes it is worthy to compare results from previous executions of your
+spider. For example, we should be able to track whether the number of items
+returned is lower than previous executions, which may indicate a problem.
+
+Scrapy allows you to collect stats in the form key/value, but these stats are
+lost when the spider finishes its execution. Spidermon provides a built-in
```

Comment:
I would say those stats are set on the dash instead of lost and one have to access them explicitly via scrapinghub client. Also, stats are still accessible via spidermon `StatsMonitorMixin` but history is not available.

Reply:
However, we need to take into account that Spidermon can be used outside Scrapy Cloud. I think the current text here is quite good, and the note below about Scrapy Cloud complements this text perfectly.

Maybe we could leave this text as is, but move the note about Scrapy Cloud a bit higher, maybe right before the example (before the ‘The following example…’ paragraph).

Reply:
We can't consider that the user is running their spiders in a specific platform, so we should not include mentions to Scrapy Could (or Dash in your comment) in the main documentation (for Scrapy Cloud users, the Note block should be enough).

I moved the note block to before the sample monitor, but it didn't look good to me. I'd prefer to keep as it is now.